docs/registration.md

```diff
@@ -40,29 +39,20 @@ You should receive an email at your registered address, with a verification link
 
```

Comment:
'WELCOME New User' in the screenshot can be updated to 'WELCOME Someone Person' 
to align it with the screenshots used for Account Creation if possible.